src/jit/target.h

```diff
@@ -1595,6 +1595,7 @@ typedef unsigned short regPairNoSmall; // arm: need 12 bits
   #define REG_VAR_ORDER            REG_R9,REG_R10,REG_R11,REG_R12,REG_R13,REG_R14,REG_R15,\
                                    REG_R8,REG_R7,REG_R6,REG_R5,REG_R4,REG_R3,REG_R2,REG_R1,REG_R0,\
                                    REG_R19,REG_R20,REG_R21,REG_R22,REG_R23,REG_R24,REG_R25,REG_R26,REG_R27,REG_R28,\
+                                   REG_IP0,REG_IP1,\
```

Comment:
What is the order used for?  

The list is currently in Caller saved, Args, Callee saved, Linkage.

Having Linkage, `REG_IP0,REG_IP1`, at he end of the list seems suspicious.  These are reserved in the ABI for linkage, but they can be used transiently as temps.

From what I remember, some of the LIR nodes use the IP regs freely during code generation, because the allocator never used to allocate them.

Reply:
> What is the order used for?

It is the order in which `tryAllocateFreeReg()` looks for a register.

> Having Linkage, REG_IP0,REG_IP1, at he end of the list seems suspicious.

I was unable to find usage of them in codegen. But I was uncertain; that's why I opened #14607.

Reply:
Since they are in `RBM_INT_CALLEE_TRASH` , it seems Ok to add them to REG_VAR_ORDER

Given the current definition of `RBM_CALLEE_TRASH_NOGC` it seems VAR ORDER should be
CALLEE_TRASH & ~ CALLEE_TRASH_NOGC
CALLEE_TRASH_NOGC
RBM_CALLEE_SAVED

Since that is not the current order, this is OK.